bindings/java/src/main/java/org/apache/opendal/FileSystemOperator.java

```diff
@@ -0,0 +1,58 @@
+package org.apache.opendal;
+
+import io.questdb.jar.jni.JarJniLoader;
+
+
+public class FileSystemOperator implements Operator {
+
```

Comment:
I think we should introduce a java code formatter. But this can be done in another PR.